### PR TITLE
esb: enable BLE 2 mbps bitrate on nRF53.

### PR DESCRIPTION
**Pull request by grochu**
Quick bugfix for enabling the BLE 2 mbps bitrate.

### CODE REVIEW

**Review by lemrey — APPROVED** [review]

File: include/esb.h [hunk 1]
```diff
@@ -127,7 +127,7 @@ enum esb_bitrate {
 #endif
 	/** 1 Mb radio mode using @e Bluetooth low energy radio parameters. */
 	ESB_BITRATE_1MBPS_BLE = RADIO_MODE_MODE_Ble_1Mbit,
-#if defined(CONFIG_SOC_SERIES_NRF52X)
+#if defined(CONFIG_SOC_SERIES_NRF52X) || defined(CONFIG_SOC_SERIES_NRF53X)
```
Comment (KAGA164) [inline include/esb.h:130]: Could you use `CONFIG_SOC_NRF5340_CPUNET` for consistency instead of `CONFIG_SOC_SERIES_NRF53X` ?

**Review by KAGA164 — APPROVED** [review]

File: include/esb.h (hunk 1 above)
Comment (grochu) [reply include/esb.h]: Fixed

**Review by magnev — CHANGES_REQUESTED** [review]
The same macro update must also be added in update_radio_bitrate() function in esb.c

**Comment by grochu** [comment]
> The same macro update must also be added in update_radio_bitrate() function in esb.c

Good catch, the `#ifdef` only contains a `case` statement so it compiled without this update (and the sample still worked).

**Comment by magnev** [comment]
Looks good!
I ran all tests, including tests for 2Mbit BLE mode on nRF53.

**Review by magnev — APPROVED** [review]

**Comment by grochu** [comment]
> Looks good!
> I ran all tests, including tests for 2Mbit BLE mode on nRF53.

Thanks @magnev, that would be fantastic to get the test results for this version.

**Review by anangl — APPROVED** [review]

**Review by shantha-14 — APPROVED** [review]

**Pull request merged**